frontend/src/components/tasks/task.css

```diff
@@ -101,14 +102,13 @@
   }
 
   .task-picture {
-    height: 3.9em;
+    height: 100%;
     width: 3.9em;
     display: block;
     position: absolute;
     background-size: contain;
     z-index: 1;
-    top: 0;
+    top: 0px;
```

Comment:
Is there any reason to specify units? I thought CSS people discourage this. A zero is a zero is a zero...